staging/src/k8s.io/apiserver/pkg/storage/etcd3/lease_manager.go

```diff
@@ -36,35 +58,33 @@ type leaseManager struct {
 	prevLeaseExpirationTime time.Time
 	// The period of time in seconds and percent of TTL that each lease is
 	// reused. The minimum of them is used to avoid unreasonably large
-	// numbers. We use var instead of const for testing purposes.
-	leaseReuseDurationSeconds int64
-	leaseReuseDurationPercent float64
+	// numbers.
+	leaseReuseDurationSeconds   int64
+	leaseReuseDurationPercent   float64
+	leaseMaxAttachedObjectCount int64
+	leaseAttachedObjectCount    int64
 }
 
 // newDefaultLeaseManager creates a new lease manager using default setting.
-func newDefaultLeaseManager(client *clientv3.Client) *leaseManager {
-	return newLeaseManager(client, 60, 0.05)
+func newDefaultLeaseManager(client *clientv3.Client, config LeaseManagerConfig) *leaseManager {
+	if config.MaxObjectCount <= 0 {
```

Comment:
nit: `< 1` is a character shorter :)

Reply:
one-liner: `config.MaxObjectCount  = int64(math.Max(float64(config.MaxObjectCount), float64(defaultLeaseMaxObjectCount)))` 

practically we're likely not to hit conversion issues here due to the expectation of low numbers (or else why would we be doing this at all?)

Reply:
Thanks for note. I would like to avoid making any changes at this point as this is exactly the code we have in master.